pyproject.toml

```
@@ -198,3 +198,54 @@ typeCheckingMode = "standard"
 # that is not meant to be exported from that module. The default value for this
 # setting is "error".
 reportPrivateImportUsage = false
+
+[tool.ruff]
+# exclude = ["__init__.py"]
+line-length = 100
+target-version = "py311"
```

```suggestion
target-version = "py310"
```

Once https://github.com/tqec/tqec/pull/599#issuecomment-2955009016 will be addressed and we validated that it fixes the problem, we can go back to Python 3.10 that should be the lowest common denominator of the Python versions we support.